<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="cudnn_test" tests="1" failures="0" errors="1">
    <testcase name="cudnn_test" status="run" duration="3" time="3"><error message="exited with error code 134"></error></testcase>
    <system-out><![CDATA[WARNING: Logging before InitGoogleLogging() is written to STDERR
I0425 22:31:28.341022  6053 cudnn_util.cc:68] Running cuDNN v7.6.0 for CUDA 9.0.20 on TITAN V
[==========] Running 7875 tests from 7 test suites.
[----------] Global test environment set-up.
[----------] 3 tests from ConvolutionTest
[ RUN      ] ConvolutionTest.GetWorkspaceSize_Overflow
[       OK ] ConvolutionTest.GetWorkspaceSize_Overflow (2 ms)
[ RUN      ] ConvolutionTest.GetConvolutionDesciptor_ArrayLengthRequested_Range
[       OK ] ConvolutionTest.GetConvolutionDesciptor_ArrayLengthRequested_Range (0 ms)
[ RUN      ] ConvolutionTest.GetGroupedConvolutionForwardOutputDim
[       OK ] ConvolutionTest.GetGroupedConvolutionForwardOutputDim (0 ms)
[----------] 3 tests from ConvolutionTest (2 ms total)

[----------] 24 tests from FromFile/ConvolutionTest
[ RUN      ] FromFile/ConvolutionTest.CompareResults/NHWC_TRUE_HALF_CONFIG_Unsupported
[       OK ] FromFile/ConvolutionTest.CompareResults/NHWC_TRUE_HALF_CONFIG_Unsupported (81 ms)
[ RUN      ] FromFile/ConvolutionTest.CompareResults/NHWC_FLOAT_CONFIG_InternalError
[       OK ] FromFile/ConvolutionTest.CompareResults/NHWC_FLOAT_CONFIG_InternalError (3 ms)
[ RUN      ] FromFile/ConvolutionTest.CompareResults/NHWC_PSEUDO_HALF_CONFIG_Crash
[       OK ] FromFile/ConvolutionTest.CompareResults/NHWC_PSEUDO_HALF_CONFIG_Crash (2 ms)
[ RUN      ] FromFile/ConvolutionTest.CompareResults/CONVOLUTION_BWD_FILTER_1025x64x7x33_96x64x7x11_VALID_Incorrect
[       OK ] FromFile/ConvolutionTest.CompareResults/CONVOLUTION_BWD_FILTER_1025x64x7x33_96x64x7x11_VALID_Incorrect (60 ms)
[ RUN      ] FromFile/ConvolutionTest.CompareResults/CONVOLUTION_BWD_FILTER_NCHW_TRUE_HALF_29x2x864x1556_2x2x3x3_SAME_Fails
[       OK ] FromFile/ConvolutionTest.CompareResults/CONVOLUTION_BWD_FILTER_NCHW_TRUE_HALF_29x2x864x1556_2x2x3x3_SAME_Fails (286 ms)
[ RUN      ] FromFile/ConvolutionTest.CompareResults/CONVOLUTION_FWD_NCHW_TENSOR_OP_52x7x112x4_873x7x3x3_VALID_GetAlgo_v7
[       OK ] FromFile/ConvolutionTest.CompareResults/CONVOLUTION_FWD_NCHW_TENSOR_OP_52x7x112x4_873x7x3x3_VALID_GetAlgo_v7 (414 ms)
[ RUN      ] FromFile/ConvolutionTest.CompareResults/BlendResult
[       OK ] FromFile/ConvolutionTest.CompareResults/BlendResult (46 ms)
[ RUN      ] FromFile/ConvolutionTest.CompareResults/GroupedConvolution
[       OK ] FromFile/ConvolutionTest.CompareResults/GroupedConvolution (4 ms)
[ RUN      ] FromFile/ConvolutionTest.CompareResults/GarbageInWorkspace
F0425 22:31:30.432655  6053 cuda_util.cc:92] Check failed: OkStatus() == GetStatus(cudaFree(ptr_)) (ok vs. CUDA Runtime API error 'an illegal memory access was encountered') 
*** Check failure stack trace: ***
    @          0x191bbca  google::LogMessage::Fail()
    @          0x191bb0e  google::LogMessage::SendToLog()
    @          0x191b45f  google::LogMessage::Flush()
    @          0x191e8de  google::LogMessageFatal::~LogMessageFatal()
    @           0x46db87  nvidia_libs_test::DeviceMemory::~DeviceMemory()
    @           0x40e7c3  _ZN16nvidia_libs_test12_GLOBAL__N_114RunConvolutionEddRKSt10unique_ptrI12cudnnContextNS_6detail18CudnnHandleDeleterEERKNS_11ConvolutionERKN4absl7variantIJ25cudnnConvolutionFwdAlgo_t29cudnnConvolutionBwdDataAlgo_t31cudnnConvolutionBwdFilterAlgo_tEEE
    @           0x410a4b  nvidia_libs_test::(anonymous namespace)::ConvolutionTest_CompareResults_Test::TestBody()
    @          0x1975bcf  testing::internal::HandleSehExceptionsInMethodIfSupported<>()
    @          0x1970131  testing::internal::HandleExceptionsInMethodIfSupported<>()
    @          0x194dd94  testing::Test::Run()
    @          0x194e6f5  testing::TestInfo::Run()
    @          0x194ed97  testing::TestSuite::Run()
    @          0x195a871  testing::internal::UnitTestImpl::RunAllTests()
    @          0x197696b  testing::internal::HandleSehExceptionsInMethodIfSupported<>()
    @          0x197103b  testing::internal::HandleExceptionsInMethodIfSupported<>()
    @          0x19591a8  testing::UnitTest::Run()
    @           0x452344  RUN_ALL_TESTS()
    @           0x451a76  main
    @     0x7f49cc595830  __libc_start_main
    @           0x40d329  _start
    @              (nil)  (unknown)]]></system-out>
  </testsuite>
</testsuites>
